Formula/x/xz.rb

```diff
@@ -2,13 +2,13 @@ class Xz < Formula
   desc "General-purpose data compression with high compression ratio"
   homepage "https://tukaani.org/xz/"
   # The archive.org mirror below needs to be manually created at `archive.org`.
```

Comment:
This needs doing

Reply:
https://archive.org/details/xz-5.6.2.tar